### DISCUSSION TITLE
How do I find files that fail to import to QL library?

### DISCUSSION THREAD

**Discussion by Npa972** [quodlibet/quodlibet#3990] [Q&A] [answered]
Hi, i have a very few missing songs after importation of my music folder in QL.
This happen on a computer but not on another one (same archlinux install) with same music folder.
Is there a way to know what's happen during importation (journal).
I have 19731 titles so i can't know witch songs are missing...
Thanks

**Comment by declension** [comment, marked as answer]
Hi. How do you know it happened at all?  Different numbers of tracks?

> Is there a way to know what's happen during importation (journal).

Generally,  starting QL in a terminal with `--debug` is best to start all investigations.

Perhaps we should add an library scan log separately (see also #3466 ) but for now any logging is in there.

> I have 19731 titles so i can't know witch songs are missing...

One idea is to export a flat list of filenames (using the python console plugin) on each and use `uniq` etc to find differences

**Comment by Npa972** [comment]
Thank you !
Debug option worked perfectly and i could identify some corrupted .oga titles.
Number of tracks are now egual.
The mystery is that this titles were imported on QL on a machine but not on the other one... Same systems, same QL... It's weird.
